Automattic/jetpack
# Boost: Fix obsolete class name to fix UI issues

#### Changes proposed in this Pull Request:

* the changes in https://github.com/Automattic/jetpack/pull/21345/files introduced a change in the name of the top level class in body and therefore stopped the correct style to be applied.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
N/A

#### Testing instructions:
* Build/watch Boost assets
* Go to Boost setting page
* Make sure that the UI is correct making the comparison below:

Before Patch (incorrect):

<img width="1536" alt="Screen Shot 2021-10-19 at 2 21 07 PM" src="https://user-images.githubusercontent.com/927932/137848655-daf71be4-2058-47df-96c6-ae0386e34102.png">

After Patch (correct):

<img width="1536" alt="Screen Shot 2021-10-19 at 2 22 10 PM" src="https://user-images.githubusercontent.com/927932/137848686-f89d6d83-5473-4851-824d-767a76e280c5.png">
